tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -947,7 +947,7 @@ def get_or_create_test_window(self, protocol):
 
 class WebDriverTestharnessExecutor(TestharnessExecutor, TestDriverExecutorMixin):
     supports_testdriver = True
-    protocol_cls = WebDriverProtocol
+    require_webdriver_bidi = False
```

Comment:
This shouldn't be a class variable; we can just rely on this being defined in `__init__`.